ehitchhiking-ui/src/app/shared/components/rating/starClickMeta.ts

```diff
@@ -0,0 +1,4 @@
+export interface starClickMeta {
```

Comment:
Identifiers of interfaces, classes, enums start with capital letter

Reply:
fixed